src/test/java/com/jcabi/beanstalk/maven/plugin/GenericMojoTest.java

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright (c) 2012-2015, jcabi.com
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met: 1) Redistributions of source code must retain the above
+ * copyright notice, this list of conditions and the following
+ * disclaimer. 2) Redistributions in binary form must reproduce the above
+ * copyright notice, this list of conditions and the following
+ * disclaimer in the documentation and/or other materials provided
+ * with the distribution. 3) Neither the name of the jcabi.com nor
+ * the names of its contributors may be used to endorse or promote
+ * products derived from this software without specific prior written
+ * permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT
+ * NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+ * FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL
+ * THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.jcabi.beanstalk.maven.plugin;
+
+import java.io.File;
+import java.util.zip.ZipFile;
+import org.junit.Test;
+import org.mockito.Mockito;
+import org.yaml.snakeyaml.error.YAMLException;
+
+/**
+ * Test case for {@link AbstractBeanstalkMojo}.
+ * @author Neo Matrix (Neo.matrix@gmail.com)
+ * @version $Id$
+ */
+public final class GenericMojoTest {
+    /**
+     * Test {@link AbstractBeanstalkMojo#execute()} method.
+     * @throws Exception If something is wrong.
+     * @checkstyle ExecutableStatementCountCheck (40 lines)
+     */
+    @Test
+    public void testExecute() throws Exception {
```

Comment:
@tmjee checkout this article about test method names http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#test-method-names